metricbeat/docker-compose.yml

```diff
@@ -15,11 +15,11 @@ services:
 
   # Used by base tests
   elasticsearch:
-    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-7.10.0}-1
+    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-7.10.1}-1
```

Comment:
Shouldn't this be 7.10.2 ?

Reply:
@jsoriano should this be 7.10.2?

Reply:
After 7.10.2 is released, yes, but not before.

Reply:
this PR should fix it https://github.com/elastic/ingest-dev/pull/807